app/controllers/course/levels_controller.rb

```diff
@@ -12,4 +34,8 @@ def index #:nodoc:
   def number_levels
```

Comment:
maybe as a separate PR, call this `#load_levels` for symmetry with load_and_authorize_resource